### DISCUSSION TITLE
Limit variabel an echten Ertrag anpassen?

### DISCUSSION THREAD

**Discussion by snooopy365** [tbnobody/OpenDTU#1104] [Ideas]
Hallo,
nach eigen Tagen probieren mit Open DTU ist mir folgendes Problem/ folgender Fehler aufgefallen.
Vermutlich liegt der Fehler bei Hoymiles, oder es wurde einfach nicht bedacht.

Wenn ein Limit eingestellt wird, gilt das Limit auf allen Eingängen des Wechselrichters. 
In meinem Fall nutze ich einen HM600. 
Wenn das Limit in Open DTU auf 300W eingestellt wird. Wird String 1 und String 2 je auf 150W limitiert.
Das ganze macht in meinen Augen in Bezug auf zukünftige Projekte wenig Sinn.

Ich plane ein HM1500 mit 4 Panels als Balkonkraftwerk zu betreiben.
Dieser darf dann laut Gesetz max. 600W einspeisen. 
Wenn ich den jetzt auf 600W limitiere und habe 2 Panels pro Hausseite, verschenke ich dauerhaft Energie.
Ich reduziere jetzt per OpenDTU, durch das 600w Limit, alle 4 Strings auf 150W.
Scheint gegen Morgen die Sonne nur auf die eine Hausseite: 300W, scheint sie Abends auf die andere Hausseite: 300W.
Das macht ja überhaupt keinen Sinn.

Ist es möglich das Limit in Open DTU auf die wirkliche Ausgangsleistung umzulegen? So das es trotz 600W Limit möglich wäre z.B. 2x Strings mit je 300W zu nutzen währen die anderen beiden 0W haben?
oder sollte nur ein Panel in der Sonne sein, z.B.: 1x400W 3*67W = 600W.

Natürlich kann ich das per MQTT Regelung extern steuern, aber wäre natürlich super wenn das Open DTU automatisch kann.
Dadurch wäre Open DTU auch wer legal 600W einspeisen will eine viel bessere Wahl als ein Wechselrichter bei dem alle Strings begrenzt sind.

Besten Dank :)

**Comment by ituri** [comment]
Guck dir mal [diese](https://github.com/tbnobody/OpenDTU/discussions/811) Diskussion an. Das beschreibt das von dir genannte Problem und eine mögliche Lösung mit dem HM-1500.
